plugins/lookup/hashi_vault.py

```diff
@@ -132,6 +132,7 @@
         - approle
         - aws_iam_login
         - jwt
+        - none
       default: token
     return_format:
       description:
```

Comment:
@jlrgraham23 I agree with @Akasurde , a note should be added saying that the `none` method was added in version `$next`. Before the plugin was moved to a collection, which sort of reset its versioning, there were other notes like this when they were relevant. You can see such an example here: https://github.com/ansible-collections/community.hashi_vault/commit/9841595d9017755b5449b144470c078f5276c503#diff-c47dc8233f99f31fc4a283e177d047b83d3df18d8e52f25a3ca3b375bffce0e9R47